src/metrics/crossref/citations.py

```diff
@@ -0,0 +1,75 @@
+from os.path import join
+import requests
+import requests_cache
+from metrics import utils
+from metrics import models
+from django.conf import settings
+from datetime import timedelta
+import logging
+
+from xml.dom.minidom import parseString
+
+LOG = logging.getLogger(__name__)
+
+requests_cache.install_cache(**{
```

Comment:
yeah, so I was confused when I saw that as well. I forgot to add a placeholder to the dir where pmc stores it's cache